_posts/2024-XX-XX-quarkus-kiota.adoc

```diff
@@ -0,0 +1,160 @@
+---
+layout: post
+title: 'The road to generated SDKs with Kiota lands to Quarkus'
+date: 2024-XX-XX
+tags: release
+synopsis: 'Introducing a Quarkus-Kiota extension and available Java tools to work with Kiota.'
+author: andreaTP
+---
+
+== The Challenge
+:imagesdir: assets/images/posts/kiota
+
+Quarkus applications typically expose functionalities through API endpoints; here, we discuss how to easily consume those APIs and provide a smooth experience for our beloved users. While HTTP calls are easy to perform in any programming language and environment, when API complexity increases, and the number of available endpoints becomes uncountable, the need for better tools becomes apparent.
+
+image::api_monster.jpeg[API monster]
+
+=== Developer Experience Wishlist:
+
+- Fully typed programmatic APIs that make discovering and invoking HTTP endpoints safe, expressive, and elegant
+- An easy and idiomatic way to plug in various authentication mechanisms
+- Smooth upgrades of the generated code when the description evolves
+
+Looking at the learnings from big companies facing this issue at scale, a viable solution is to ship language-specific SDKs (Software Development Kits). For years, companies have been paying developers to manually provide the abstraction layer, making cloud services functionalities more easily accessible. This approach adds up to the maintenance burden, increases the operational complexity of releases, and, worst, is tedious for developers.
+
+To pave the way for better tooling, new standards are being defined in the computer industry. For building HTTP-based APIs, link:https://www.openapis.org[OpenAPI] stands out as a widely adopted option.
+In the context of this article we are going to assume you already have an OpenAPI description of the service.
+
+=== A Story of Open Source Collaboration
```

Comment:
I love both Microsoft and Red Hat - but they are big corporations and its not like all of red hat and all of Microsoft did this  - could you highlight bit more specifically what parts of those companies got involved ?